apps/cowswap-frontend/src/legacy/state/orders/utils.ts

```diff
@@ -319,7 +319,7 @@ export function getEstimatedExecutionPrice(
 
   // When fee > amount, return 0 price
   if (!remainingSellAmount.greaterThan(ZERO_FRACTION)) {
-    return new Price(inputToken, outputToken, '0', '0')
+    return new Price(inputToken, outputToken, '0', '1')
```

Comment:
This was dumb, creating a fraction 0/0.
No wonder why there are so many division by 0 errors.
Who was the idiot that wrote that? 🙈